server/controllers/medicationController.js

```diff
@@ -70,6 +70,26 @@ exports.getListOfMedicationUserComments = function(req, res) {
     }
 };
 
+exports.getListOfRemovedMedicationUserComments = function(req, res) {
```

Comment:
Q over the name of this method. Other methods in the PR refer to MedicationComments - would it be better to keep this consistent, i.e. getListOfRemovedMedicationComments?